vendor/assets/stylesheets/rolodex/settings/utilities/lib/_borders.sass

```diff
@@ -0,0 +1,87 @@
+// Border
+// ========================================
+
+$border-color: $gray-light
+$border-style: solid
+$border-width: 1px
+
+// Default border styling
+
+.b-all
+  border-color: $border-color
+  border-style: $border-style
+  +rem(border-width, $border-width)
+
+.b-top
+  border-top-style: $border-style
+  +rem(border-top-width, $border-width)
+  border-color: $border-color
+
+.b-right
+  border-right-style: $border-style
+  +rem(border-right-width, $border-width)
+  border-color: $border-color
+
+.b-bottom
+  border-bottom-style: $border-style
+  +rem(border-bottom-width, $border-width)
+  border-color: $border-color
+
+.b-left
+  border-left-style: $border-style
+  +rem(border-left-width, $border-width)
+  border-color: $border-color
+
+.b-0,
+.b-none
+  border: 0
+
+// State
+
+.b-active
+  border-color: $blue
+
+.b-offset
+  border-color: $white-offset
+
+.b-gray
+  border-color: $gray
+
+// Border radius
+
+.br-circle
+  border-radius: 50%
+
+.br-all
+  +rem(border-radius, $border-radius-base)
+
+.br-mini
+  +rem(border-radius, $border-radius-mini)
+
+.br-0,
+.br-none
+  border-radius: 0
+
+.br-top
+  +rem(border-radius, $border-radius-base $border-radius-base 0 0)
```

Comment:
I'd break this out into long hand, using two properties. 

It'd be rare but there could be cases where you want one border radius on the top and another on the bottom, etc.